pyproject.toml

```
@@ -56,6 +57,7 @@ test = [
     "threadpoolctl",
 ]
 test_all = [
+    "astropy[all]",  # installs all optional run-time dependencies
     "astropy[test]",  # installs the [test] dependencies
```

The first comment is useful (because it mentiones "run-time"), the second comment is not. The useless comment isn't added by this pull request, but several analogous and equally useless comments are being introduced that should all be cleaned up.
```suggestion
    "astropy[all]",  # installs all optional run-time dependencies
    "astropy[test]"
```

---

Done.